From b54a87d9f7aba5ed99802bc3b25473eb2dda2527 Mon Sep 17 00:00:00 2001
From: Grace Cham <hscham@chromium.org>
Date: Wed, 13 Jul 2022 23:11:33 +0900
Subject: [PATCH] forward compatibility: Alias bit_cast to base::bit_cast [tmp]

The function is moved into base:: namespace in r998789
(crrev.com/c/3615258).

Change-Id: Id62dcd469d3727724c93a0cd3658d96baf65d481
---
 base/bit_cast.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/base/bit_cast.h b/base/bit_cast.h
index a8e0da7dd4..43f8c01311 100644
--- a/base/bit_cast.h
+++ b/base/bit_cast.h
@@ -45,4 +45,8 @@ inline
 
 }  // namespace base
 
+template <class Dest, class Source>
+[[deprecated("The function was moved, use base::bit_cast.")]]
+inline Dest bit_cast(Source source) { return base::bit_cast<Dest>(source); }
+
 #endif  // BASE_BIT_CAST_H_
-- 
2.37.1.595.g718a3a8f04-goog

